input.lisp

```diff
@@ -184,8 +184,8 @@ Available completion styles include
 ;;; keysym functions
 
 (defun is-modifier (keycode)
-  "Return t if keycode is a modifier"
-  (or (find keycode *all-modifiers* :test 'eql)
+  "Return T if KEYCODE is a modifier."
+  (or (member keycode *all-modifiers* :test 'eql)
```

Comment:
Is there a reason for this change? Is `member` maybe measurably faster than `find`?  Other than that, I'd find it more useful to get the keycode back than the tail of `*all-modifiers*, assuming that the “No Symbol” alternative is removed here.

Reply:
> Is there a reason for this change?

Not a strong one just that semantically I care about membership

> Is member maybe measurably faster than find?

I wouldn't expect member to be faster. They both have to traverse the list linearly and upon find it they return. If anything member would return 'more' items and it could put more pressure on the GC, but I doubt there be any difference

:pray: I'll revert this change

Reply:
I understand.  For me, the semantic difference between “find” and “member” is not that big (but I'm not a native English speaker).

Thanks for your consideration.  I'm just adding my thoughts, but I do not want to imply that I'd know better than you (and you do all the work here).

Reply:
> I'm just adding my thoughts, but I do not want to imply that I'd know better 

Misunderstandings are more likely to happen when communicating over text. I'm sorry if what I wrote gave you the impression that your comment wasn't received. I'm actually grateful when you, or anyone else for that matter, chimes in and reviews a pull request.

Reply:
In general, I prefer performance discussions to be framed around empirical evidence that the code in question is a hotspot.  Thanks for taking the time to review this PR!